extensions/amazon-aws/src/components/cloudwatch/CloudwatchLogStreams.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from "react";
+import { Action, ActionPanel, List } from "@raycast/api";
+import { useCachedPromise } from "@raycast/utils";
+import { fetchLogStreams, getTaskCWLogsGroupUrl } from "../../actions";
+import { getActionOpenInBrowser, getActionPush, getFilterPlaceholder } from "../../util";
+import { LogStartTimes } from "../../interfaces";
+import CloudwatchLogs from "./CloudwatchLogs";
+import CloudwatchLogsTimeDropdown from "../searchbar/CloudwatchLogsTimeDropdown";
+
+function CloudwatchLogStreams({ logGroupName }: { logGroupName: string }) {
+  const [logStartTime, setLogStartTime] = useState<LogStartTimes>(LogStartTimes.OneHour);
+  const { data: streams, isLoading } = useCachedPromise(fetchLogStreams, [logGroupName], { keepPreviousData: true });
+
+  return (
+    <List
+      isLoading={isLoading}
+      searchBarPlaceholder={getFilterPlaceholder("stream")}
+      searchBarAccessory={<CloudwatchLogsTimeDropdown logStartTime={logStartTime} onChange={setLogStartTime} />}
```

Comment:
Could you explain me the use case of this time dropdown?
I think a user might expect this to filter the log streams list, but instead, it is only used to change the Open In Browser action, correct?

Reply:
It is for the final view logs `Detail` view. As you cannot encompass a `Detail` in a `List` I had to add this on the screen before the logs `Detail` view, otherwise the period to be retrieved cannot be configured.